tests/kvasir-tests/Makefile.common

```diff
@@ -121,6 +121,9 @@ endif
 endif
 
 ifeq ($(OS_ID),Ubuntu)
+ifeq ($(OS_REL),24.04)
+	GOAL := goal-Ubuntu2404
+endif
 ifeq ($(OS_REL),23.10)
```

Comment:
Should this and some other `ifeq` statements be removed, since the test for "24.04" now exists?

Reply:
A few goal files differ between Ubuntu 22.04 and 24.04; so I think we need to keep this change for now.  We can review again when we decide what to do about the other, stale goal files.